src/api/app/models/kiwi/image.rb

```diff
@@ -103,6 +125,10 @@ def outdated?
 
     package.kiwi_image_outdated?
   end
+
+  def packages
+    package_groups.where(kind: 'image', pattern_type: 'onlyRequired').map(&:packages).flatten
```

Comment:
You can add conditions to an association here is an example from the [docs](http://api.rubyonrails.org/classes/ActiveRecord/Associations/ClassMethods.html)
```ruby
has_many :approved_comments, -> { where(approved: true) }, class_name: 'Comment'
```

But I wonder, is packages the right name for this if its only returning packages that are of type image and pattern_type onlyRequired? Maybe the association should be named "required_image_packages"? I dont really know the context of this stuff so I might be wrong..